PterodactylLib/Classes/Requests.swift

```diff
@@ -26,3 +26,9 @@ struct UpdateDefaultsRequest: Codable {
     let appBundleId: String
     let defaults: [String: UpdateDefaultsValue]
 }
+
+struct DeleteDefaultsRequest: Codable {
+    let simulatorId: String
+    let appBundleId: String
+    let keys: [String]
+}
```

Comment:
This is the structure sent between the library and the server for deleting user defaults keys.